README.md

```diff
@@ -2,6 +2,18 @@
 * docker and docker-compose is installed
 * IPV4 forwarding is enabled - net.ipv4.ip_forward=1
 * a [dockerhub](https://hub.docker.com/) account with access to the bmsapp repo. For external dev teams access requests can be performed via our [HelpDesk](https://ibplatform.atlassian.net/servicedesk/customer/portal/4/group/30/create/60)
+
+## Configurations
+* **.env.example** - this is where we setup environment variables utilized by the docker compose configuration. In order to use it it must be renamed to .env
+* **config** - this contains the configuration files for the BMSScripts inside bmsapp.
```

Comment:
Change to `DbScripts` (instead of BMSScripts)

Reply:
ok